Baikal/Kernels/CL/bxdf_uberv2.cl

```diff
@@ -70,9 +70,9 @@ void UberV2_ApplyShadingNormal(
 
     if ((layers & kShadingNormalLayer) == kShadingNormalLayer)
     {
-        dg->n = normalize(shader_data->shading_normal.z * dg->n + shader_data->shading_normal.x * dg->dpdu + shader_data->shading_normal.y * dg->dpdv);
-        dg->dpdv = normalize(cross(dg->n, dg->dpdu));
-        dg->dpdu = normalize(cross(dg->dpdv, dg->n));
+        dg->n = normalize(shader_data->shading_normal.z * dg->n + shader_data->shading_normal.x * dg->tangent + shader_data->shading_normal.y * dg->bitangent);
```

Comment:
add comment Why dpdu/dpdv was replaced with tangent/bitangent

Reply:
I've added a note about this in Scene_FillDifferentialGeometry